spec/PeriodicBackgroundSync-index.bs

```diff
@@ -107,36 +107,39 @@ A <dfn>state</dfn>, which is one of "`pending`", "`firing`", "`suspended`" or "`
 The [=periodic sync scheduler=] is responsible for scheduling firing of [=periodicsync events=].
 In response to these triggers, the scheduler either schedules delayed processing to fire a [=periodicsync event=] at the appropriate time in the future, or cancels such scheduling.
 
-  <div dfn-for="periodic sync scheduler">
-    The scheduler maintains:
-    * The <dfn>time of last fire</dfn>, a [=map=], where each key is an [=origin=], and each item is a timestamp. The keys of this map are the [=origin|origins=] associated with the [=periodic sync registration/service worker registration|service worker registrations=] of [=active periodic sync registrations=]. This is initially an empty [=map=].
-  </div>
+<div dfn-for="periodic sync scheduler">
+  The scheduler maintains the <dfn>time of last fire</dfn>, a [=map=], where each key is an [=origin=], and each item is a timestamp. The keys of this map are the [=origin|origins=] associated with the [=periodic sync registration/service worker registration|service worker registrations=] of [=active periodic sync registrations=]. This is initially an empty [=map=].
+</div>
 
-The <dfn>effective minimum sync interval for origin</dfn> |origin| (an [=origin=]), is the [=minimum periodic sync interval for any origin=] + some user agent defined amount for the |origin|.
+The <dfn>effective minimum sync interval for origin</dfn> |origin| (an [=origin=]), is the [=minimum periodic sync interval for any origin=] + some user agent defined amount for the |origin| (which may be `infinite`). The user agent defined amount MUST be used to mitigate the concerns raised in [[#privacy]], [[#security]] and [[#resources]].
 
-Note: The user agent defined amount could be based off the amount of engagement the user has with the origin. This value can be different each time [=effective minimum sync interval for origin=] is called.
+Note: The user agent defined amount can be based on the amount of engagement the user has with the origin. This value can be different each time [=effective minimum sync interval for origin=] is called. An `infinite` value suspends firing any [=periodicsync event|periodicsync events=].
```

Comment:
```suggestion
Note: The user agent defined amount can be based on the amount of engagement the user has with the origin. This value can be different each time [=effective minimum sync interval for origin=] is called. No [=periodicsync event|periodicsync events=] will fire for a particular origin if the last value returned by [=effective minimum sync interval for origin=] is `Infinity`.
```

I've tried to reword the above to make it clear that origins can come back from `Infinity` to some finite value. I don't know if I've achieved that.

Reply:
Thanks! updated.